spec/support/verifiable.rb

```diff
@@ -179,4 +179,66 @@
     end
   end
 
+  describe "#overrided_methods" do
+
+    let(:user) {create(:user)}
+
+    before do
+      Setting["feature.user.skip_verification"] = 'active'
+    end
+
+    it "residence_verified? is true if overrided" do
+      expect(user.residence_verified?).to eq(true)
+    end
+
+    it "sms_verified? is true if overrided" do
+      expect(user.sms_verified?).to eq(true)
+    end
+
+    it "level_two_verified? is true if overrided" do
+      expect(user.level_two_verified?).to eq(true)
+
+      user.update(residence_verified_at: Time.current)
+      expect(user.level_two_verified?).to eq(true)
+
+      user.update(confirmed_phone: "123456789", residence_verified_at: false)
+      expect(user.level_two_verified?).to eq(true)
+    end
+
+    it "level_three_verified? is true  if overrided" do
+      expect(user.level_three_verified?).to eq(true)
+    end
+
+    it "unverified? is true if overrided" do
```

Comment:
This subject is not consistent with the expectation! It says it should be `true` but the expectation wants a `false`... also `unverified?` method is NOT affected but the skip verification :/ Please read carefully code changes before commiting, and after opening the PR. Reviewers time is really important having to read tons of PR's, you only have to read one at a time before moving into the next issue.